app/main.py

```diff
@@ -1,52 +1,36 @@
-import json
-import xml.etree.ElementTree as ET
-
-
 class Book:
-    def __init__(self, title: str, content: str):
+    def __init__(self, title: str, content: str) -> None:
         self.title = title
         self.content = content
```

Comment:
1 blank line before and after class methods. Also do you think it's a good idea to create classes in the main file of your project?)